range.hh

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (C) 2015 ScyllaDB
+ * Copyright (C) 2020 ScyllaDB
```

Comment:
Updated the wrong file? Or did you mean to update it. If the latter I'm curious as to why, I didn't see copyright year updated in existing source files before.

Reply:
This is a new file, with backward-compatible name and content.